cmd/podmanV2/containers/start.go

```diff
@@ -0,0 +1,88 @@
+package containers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/containers/libpod/cmd/podmanV2/utils"
+
+	"github.com/containers/libpod/cmd/podmanV2/common"
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/libpod/define"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	startDescription = `Starts one or more containers.  The container name or ID can be used.`
+	startCommand     = &cobra.Command{
+		Use:     "start [flags] CONTAINER [CONTAINER...]",
+		Short:   "Start one or more containers",
+		Long:    startDescription,
+		RunE:    start,
+		PreRunE: preRunE,
+		Args:    cobra.MinimumNArgs(1),
+		Example: `podman start --latest
+  podman start 860a4b231279 5421ab43b45
+  podman start --interactive --attach imageID`,
+	}
+)
+
+var (
+	startOptions entities.ContainerStartOptions
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode},
+		Command: startCommand,
+	})
+	flags := startCommand.Flags()
+	flags.BoolVarP(&startOptions.Attach, "attach", "a", false, "Attach container's STDOUT and STDERR")
+	flags.StringVar(&startOptions.DetachKeys, "detach-keys", common.GetDefaultDetachKeys(), "Select the key sequence for detaching a container. Format is a single character `[a-Z]` or a comma separated sequence of `ctrl-<value>`, where `<value>` is one of: `a-z`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`")
+	flags.BoolVarP(&startOptions.Interactive, "interactive", "i", false, "Keep STDIN open even if not attached")
+	flags.BoolVarP(&startOptions.Latest, "latest", "l", false, "Act on the latest container podman is aware of")
+	flags.BoolVar(&startOptions.SigProxy, "sig-proxy", false, "Proxy received signals to the process (default true if attaching, false otherwise)")
+	if registry.IsRemote() {
+		_ = flags.MarkHidden("latest")
+		_ = flags.MarkHidden("sig-proxy")
+	}
+
+}
+
+func start(cmd *cobra.Command, args []string) error {
+	var errs utils.OutputErrors
+	if len(args) > 1 && startOptions.Attach {
```

Comment:
Should move this into ContainerStart IMO

Reply:
i like the idea of catching this right away.  why proceed at all under invalid conditions?

Reply:
Fair enough